index.html

```diff
@@ -73,7 +77,7 @@
 
   <div id="map"></div>
 
-  <script data-spreadsheet="0Agcr__L1I1PDdEpoMnhxR0RHdkFsWlFtNTlEZlltR0E" src="js/script.js"></script>
+  <script data-spreadsheet="0Agcr__L1I1PDdEpoMnhxR0RHdkFsWlFtNTlEZlltR0E" data-actionkit="peoples-climate-plan-actions|break-free-world" src="js/script.js"></script>

```

Comment:
@ejucovy 

This is where you would specify which campaigns to show. Is there a specific one you want us to list here? Otherwise, I'm assuming that once this is merged, it will go live.